schema/material/interface.json

```diff
@@ -0,0 +1,69 @@
+{
+    "$id": "interface",
+    "$schema": "http://json-schema.org/draft-07/schema#",
+    "title": "Interface Schema",
+    "type": "object",
+    "allOf": [
+        {
+            "$ref": "../material.json"
+        }
+    ],
+    "properties": {
+        "interface_config": {
+            "type": "object",
+            "properties": {
+                "type": {
+                    "type": "string",
+                    "enum": ["interface"]
+                },
+                "source": {
+                    "type": "object",
+                    "properties": {
+                        "slab1_config": {
+                            "$ref": "slab.json"
+                        },
+                        "slab2_config": {
+                            "$ref": "slab.json"
+                        }
```

Comment:
Maybe we still need to refer to them as Substrate and Film, since this separation is used for straining